### ISSUE TITLE
dbus-cgwacs: Cannot write /Position on dbus

### ISSUE THREAD

**Issue by izak** [victronenergy/venus#1389]
Not a serious problem yet. You can still change the position using the Energy Meters menus, as in the past. 

This works when compiling the binary on a desktop, but fails on the GX. And therefore also from the Setup menu item in the GUI, ie this is broken:

![Selection_202](https://github.com/user-attachments/assets/c78a1a95-9fd8-4be2-a3f2-43ef5c48a027)

Tested on v3.33 and v3.50. Seems to be broken for a while already.

**Comment by izak** [comment]
This is a very strange bug. It seems to be related to our SDK.

I was mistaken that it is broken in Venus 3.50. It was broken with a version I compiled locally, and I never suspected compiler issues to be at the root. It also appears to be broken with Venus 3.33, but that is what it is.

It works correctly after replacing the original binary from 3.50, and also with Venus 3.52.

The issue is because our SDK uses this mkspec:

```
QMAKESPEC=".../sysroots/cortexa8hf-neon-ve-linux-gnueabi/usr/share/qtopia/mkspecs/linux-g++"
```

This causes some of the metatype stuff to be broken, and a setProperty call on a property that has a enum as type fails. That's also why it works correctly on the desktop, it uses a different spec file.

In short, we have to be careful of binaries we compile using the SDK. It uses a lesser version of QT in which (apparently) not everything is supported. For example, the Q_ENUM macro which has been in QT since v5.5, is not in the QTopia spec.

Closing this issue, as nothing needs doing right now.
